wdtk-wikibaseapi/src/main/java/org/wikidata/wdtk/wikibaseapi/StatementUpdate.java

```diff
@@ -189,6 +189,16 @@ public String getJsonUpdateString() {
 		}
 		return updatedStatements;
 	}
+	
+	/**
+	 * Returns true when the edit is not going to change anything on the item.
+	 * In this case, the change can be safely skipped, except if the side effects
+	 * of a null edit are desired.
+	 */
+	@JsonIgnore
+	public boolean isNull() {
```

Comment:
A method name like "isNullEdit" or "isEmptyEdit" is maybe more meaningful. Here it looks a lot like a method to check if something == null.

Reply:
makes sense, I will change that.